ql/src/test/org/apache/hadoop/hive/ql/io/TestBucketCodec.java

```diff
@@ -77,13 +77,13 @@ public void testGetBucketCodecVersion1() {
 
   @Test(expected = IllegalArgumentException.class)
   public void testGetBucketCodecVersion1EncodeNegativeBucketId() {
-    BucketCodec.getCodec(1).encode(new AcidOutputFormat.Options(null).bucket(-1).statementId(16));
+    BucketCodec.getCodec(1).encode(new AcidOutputFormat.Options(null).bucket(-1).statementId(16).maxStmtId(BucketCodec.MAX_STATEMENT_ID));
```

Comment:
Why is this needed? This is the default behaviour if maxstatementId was not set, no?

Reply:
Right, this is no longer needed. I removed it.